docs/NewOBOFC

```diff
@@ -0,0 +1,20 @@
+---
+layout: doc
+id: OFOCMember
+title: Becoming a member of the OBO Operations Committee
+---
+
+# Becoming a member of the OBO Foundry Operations Committee (OFOC)
+
+Note: This is _not_ a formal process, but rather a piece of documentation to make the current _informal_ process more transparent. 
+
+1. If someone (applicant) is interested in becoming a member of the operations group, they should contact an existing group member, 
+who explains to them that the various duties expected from members of the OFOC (e.g. chairing calls, addressing issues on GitHub, joining a working group).
+2. The applicant confirms they are willing to take up some of these duties.
+3. The current members are then asked via an email on obo-operations if anyone objects to adding the candidate member. 
+4. If conflicts/concerns arise, we try to first find consensus at the next OFOC call. 
+5. In cases where no consensus can be reached, we ask for a formal vote with the majority of active members
+
```

Comment:
Should we add a:
6. New member is directly emailed a membership acceptance message from the existing group member (or other operations group member).  
It would be nice to have a few line template for that email, including invite links to EditorialWG.html etc. and name of OBO operations list to sign up to.

Reply:
Added this.